azure/README.md

```diff
@@ -116,20 +116,48 @@ $ terraform init
 $ terraform apply
 ```
 
+In the output of `terraform apply` command, the `cosmosdb_account_endpoint` and the `cosmosdb_account_primary_master_key` are used to configure Scalar DL in the next section. Please take a note of them.
```

Comment:
The sentence sounds a bit weird. You mean this?
```suggestion
In the output of `terraform apply` command, you will see  `cosmosdb_account_endpoint` and `cosmosdb_account_primary_master_key`. Please take note of them since they are used to configure Scalar DL in the next section. 
```